test/cli/cli.test.js

```diff
@@ -20,6 +20,23 @@ describe('CLI', () => {
       .then((output) => {
         expect(output.code).toEqual(0);
         expect(output.stderr.includes('0% compiling')).toBe(true);
+        // should not profile
+        expect(
+          output.stderr.includes('ms after chunk modules optimization')
+        ).toBe(false);
+        done();
+      })
+      .catch(done);
+  });
+
+  it('--progress --profile', (done) => {
+    testBin('--progress --profile')
+      .then((output) => {
+        expect(output.code).toEqual(0);
+        // should profile
+        expect(
+          output.stderr.includes('ms after chunk modules optimization')
```

Comment:
I will try, last time output did not always match though

Reply:
We can implement helper for tests what remove unnecessary stuff from output to keep snapshot small and testable

Reply:
> We can implement helper for tests what remove unnecessary stuff from output to keep snapshot small and testable

I will try that.

> No need !! we already validate this only as boolean, so no need convert to boolean again

It could be undefined though.

Reply:
Thanks, you are right :+1: